packages/fluentui/react-northstar/src/themes/teams/getBorderFocusStyles.ts

```diff
@@ -9,12 +9,18 @@ type BorderFocusStyles = CSSBorderStyles & {
     | {
         borderWidth: string;
         borderRadius: string;
+        focusBorderRadius?: string;
         focusInnerBorderColor: string;
         focusOuterBorderColor: string;
+        focusInnerBorderWidth?: string;
+        focusOuterBorderWidth?: string;
```

Comment:
do you intend to set these variables in site variables in this PR too ?

Reply:
@ling1726 no, because they are optional properties and change should be backward compatible. 
The value for this properties are introduced in theme's override in the project, not in library.

Reply:
I see, in this case perhaps there should be a screener test to verify that these overrides work, since they will never be caught by any other of the VR tests because we will never use them in any other situation.